sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/PanacheEndpoint.java

```diff
@@ -0,0 +1,135 @@
+package io.quarkus.ts.reactive;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@Path("/library")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class PanacheEndpoint {
+
+    @Inject
+    AuthorRepository authors;
+
+    @GET
+    @Path("books")
+    public Multi<String> all() {
+        return Book.all()
+                .map(Book::getTitle);
+    }
+
+    @GET
+    @Path("books/{id}")
+    public Uni<Response> find(Integer id) {
```

Comment:
```suggestion
    public Uni<Response> findById(Integer id) {
```

Reply:
Again, why? We never call this method directly, and usage of id is easily seen from the method signature and URl template

Reply:
It's just for readability purposes, you are searching "by" some criteria, you are not trying to find integer